terraform/environments/mlra/module/cloudwatch/main.tf

```diff
@@ -0,0 +1,290 @@
+resource "aws_cloudwatch_metric_alarm" "esccpuoverthreshold" {
+  alarm_name         = "ECS-CPU-high-threshold-alarm"
+  alarm_description  = "If the CPU exceeds the predefined threshold, this alarm will trigger. \n Please investigate."
+  namespace          = "AWS/ECS"
+  metric_name        = "CPUUtilization"
+  statistic          = "Average"
+  period             = "60"
+  evaluation_periods = "5"
+  threshold          = var.pECSCPUAlarmThreshold
+  treat_missing_data = "breaching"
+  dimensions = {
+    ClusterName = var.pClusterName
+  }
+  comparison_operator = "GreaterThanThreshold"
+}
+
+resource "aws_cloudwatch_metric_alarm" "ecsmemoryoverthreshold" {
+  alarm_name         = "${var.appnameenv}-ECS-Memory-Over-Threshold"
```

Comment:
Could you apply this alarm name format to the rest of the alarms please?

Reply:
all alarm names have been updated